### ISSUE TITLE
Total Score Display?

### ISSUE THREAD

**Issue by gwenbeebe** [COHHIO/Rminor_elevated#59]
Is it intentional that we aren't currently displaying the total estimated score on the BoS CoC Competition tab, or had it just not been added? I was thinking it could make sense either as a new bottom row or up top with the program name in parentheses, like "Project Name (estimated score xx out of yy)." That could help us and providers with easier at-a-glance comparisons, if that's something we want to support. What do you think?

**Comment by kiadso** [comment]
Yes- displaying the total score is something we should be doing, I don't know why I never did this!

**Comment by gwenbeebe** [comment]
You got it! Do you have a location preference for displaying it?
